lib/util.ts

```diff
@@ -33,10 +32,10 @@ export function filterStackTrace(text: string): string {
  * Internal helper for abstraction of polymorphic filenameOrFn properties.
  * @param {object} filenameOrFn The filename or function that we will execute.
  * @param {Array.<object>}} args The args to pass into filenameOrFn.
- * @return {q.Promise} A promise that will resolve when filenameOrFn completes.
+ * @return {Promise} A promise that will resolve when filenameOrFn completes.
  */
 export function runFilenameOrFn_(configDir: string, filenameOrFn: any, args?: any[]): Promise<any> {
-  return Promise((resolvePromise) => {
+  return new Promise((resolvePromise) => {
```

Comment:
nit: How about just `resolve` here instead of `resolvePromise` (I know that was the old style but I prefer simplifying it while we are in here)